WcaOnRails/app/views/layouts/_navigation.html.erb

```diff
@@ -56,6 +56,7 @@
             <% end %>
             <li class="divider"></li>
             <li><a href="/results/statistics.php"><%= icon('area-chart') %> <%= t '.statistics' %></a></li>
+            <li><a href="<%= relations_path %>"><%= icon('sitemap') %> <%= t '.relations' %></a></li>
```

Comment:
@Laura-O @SAuroux As you're so convinced about it, I'll surely remove it from the navbar and try to find some better place to have this link somewhere (maybe the statistics page) =)